### PR TITLE
[Lottie Exporter] Added documentation for gradient layers

### PR DESCRIPTION
**Pull request by saurabh-ramola**
Added documentation for Linear Gradient Layer and Radial Gradient Layer.

@AnishGG is this correct?

### CODE REVIEW

File: docs/projects/lottie/layers/linear_gradient.rst [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+=====================
+
++---------------------+---------------------------------------------------------------------------+
+|       Property      |                         Path to Property in lottie                        |
++=====================+===========================================================================+
+|       Z depth       |               Depends on ordering of layers in lottie format              |
++---------------------+---------------------------------------------------------------------------+
+|        Amount       |   layers/shape.json -> "ef" -> effects/gFill.json -> effects/opacity.json |
++---------------------+---------------------------------------------------------------------------+
+|     Blend_method    |Not supported                                                              |
```
Comment (AnishGG) [inline docs/projects/lottie/layers/linear_gradient.rst:11]: I guess this you have supported. Please check again.

File: docs/projects/lottie/layers/linear_gradient.rst [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
++---------------------+---------------------------------------------------------------------------+
+|        Gradient     |    layers/shape.json -> "ef" -> effects/gfill.json                        |
++---------------------+---------------------------------------------------------------------------+
+|       Point 1       |layers/shape.json ->  "ef" -> effects/gFill.json ->  "s"                   |
```
Comment (AnishGG) [inline docs/projects/lottie/layers/linear_gradient.rst:15]: There is no param point1 or point2. Only `p1` and `p2` are there. I see that you have added point1 and point2 params, make a NOTE of these 2 things and why have you added these 2.

File: docs/projects/lottie/layers/radial_gradient.rst [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+=====================
+
++---------------------+---------------------------------------------------------------------------+
+|       Property      |                         Path to Property in lottie                        |
++=====================+===========================================================================+
+|       Z depth       |               Depends on ordering of layers in lottie format              |
++---------------------+---------------------------------------------------------------------------+
+|        Amount       |   layers/shape.json -> "ef" -> effects/gFill.json -> effects/opacity.json |
++---------------------+---------------------------------------------------------------------------+
+|     Blend_method    |Not supported                                                              |
```
Comment (AnishGG) [inline docs/projects/lottie/layers/radial_gradient.rst:11]: Same as previous blend method. I guess you have supported this.

File: docs/projects/lottie/layers/radial_gradient.rst [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
++---------------------+---------------------------------------------------------------------------+
+|        Gradient     |      layers/shape.json -> "ef" -> effects/gfill.json                      |
++---------------------+---------------------------------------------------------------------------+
+|       Center        |      layers/shape.json ->  "ef" -> effects/gFill.json ->  "s"             |
```
Comment (AnishGG) [inline docs/projects/lottie/layers/radial_gradient.rst:15]: Show how you used center and radius as you have mentioned in the comments in the code: https://github.com/synfig/synfig/pull/1446/files#diff-5a70b0c8e040579359b43c0cfd55b9f9R69

**Comment by saurabh-ramola** [comment]
> @saurabh-ramola Have a look at the changes requested.

I have incorporated the changes as requested by you. ^__^

File: docs/projects/lottie/layers/radial_gradient.rst (hunk 4 above)
Comment (saurabh-ramola) [reply docs/projects/lottie/layers/radial_gradient.rst:15]: I have added the description for the calculation of gradient start point and gradient end point in the important points section.

File: docs/projects/lottie/layers/linear_gradient.rst [hunk 5]
```diff
@@ -0,0 +1,27 @@
+Linear Gradient Layer
+=====================
+
++---------------------+---------------------------------------------------------------------------+
+|       Property      |                         Path to Property in lottie                        |
++=====================+===========================================================================+
+|       Z depth       |               Depends on ordering of layers in lottie format              |
++---------------------+---------------------------------------------------------------------------+
+|        Amount       |   layers/shape.json -> "ef" -> effects/gFill.json -> effects/opacity.json |
```
Comment (AnishGG) [inline docs/projects/lottie/layers/linear_gradient.rst:9]: There is not path effects/gFill.json. Please check this.

File: docs/projects/lottie/layers/linear_gradient.rst [hunk 6]
```diff
@@ -0,0 +1,27 @@
...
++---------------------+---------------------------------------------------------------------------+
+|     Blend_method    |Partially supported                                                        |
++---------------------+---------------------------------------------------------------------------+
+|        Gradient     |    layers/shape.json -> "ef" -> effects/gfill.json                        |
```
Comment (AnishGG) [inline docs/projects/lottie/layers/linear_gradient.rst:13]: Same. No effects/gFill.json

File: docs/projects/lottie/layers/linear_gradient.rst [hunk 7]
```diff
@@ -0,0 +1,27 @@
...
++---------------------+---------------------------------------------------------------------------+
+|       p1            |layers/shape.json ->  "ef" -> effects/gFill.json ->  "s"                   |
```
Comment (AnishGG) [inline docs/projects/lottie/layers/linear_gradient.rst:15]: Here also

File: docs/projects/lottie/layers/linear_gradient.rst [hunk 8]
```diff
@@ -0,0 +1,27 @@
...
++---------------------+---------------------------------------------------------------------------+
+|       p2            |layers/shape.json ->  "ef" -> effects/gFill.json ->  "e"                   |
```
Comment (AnishGG) [inline docs/projects/lottie/layers/linear_gradient.rst:17]: Here also

File: docs/projects/lottie/layers/linear_gradient.rst [hunk 9]
```diff
@@ -0,0 +1,27 @@
...
++---------------------+---------------------------------------------------------------------------+
+|      Loop           |                               Not supported                               |
++---------------------+---------------------------------------------------------------------------+
+|      Zigzag         |                               Not supported                               |
++---------------------+---------------------------------------------------------------------------+
+
+Important points
+----------------
+
+- p1 and p2 parameters are used to calculate the gradient start point -> "s" and the gradient end point -> "e" respectively.
```
Comment (AnishGG) [inline docs/projects/lottie/layers/linear_gradient.rst:27]: Please make a note about `point1` and `point2` also, which you introduced for the rectangle layer.

File: docs/projects/lottie/layers/radial_gradient.rst [hunk 10]
```diff
@@ -0,0 +1,28 @@
+Radial Gradient Layer
+=====================
+
++---------------------+---------------------------------------------------------------------------+
+|       Property      |                         Path to Property in lottie                        |
++=====================+===========================================================================+
+|       Z depth       |               Depends on ordering of layers in lottie format              |
++---------------------+---------------------------------------------------------------------------+
+|        Amount       |   layers/shape.json -> "ef" -> effects/gFill.json -> effects/opacity.json |
```
Comment (AnishGG) [inline docs/projects/lottie/layers/radial_gradient.rst:9]: No path effects/gFill.json

File: docs/projects/lottie/layers/radial_gradient.rst [hunk 11]
```diff
@@ -0,0 +1,28 @@
...
++---------------------+---------------------------------------------------------------------------+
+|     Blend_method    |Partially supported                                                        |
++---------------------+---------------------------------------------------------------------------+
+|        Gradient     |      layers/shape.json -> "ef" -> effects/gfill.json                      |
```
Comment (AnishGG) [inline docs/projects/lottie/layers/radial_gradient.rst:13]: same

File: docs/projects/lottie/layers/radial_gradient.rst [hunk 12]
```diff
@@ -0,0 +1,28 @@
...
++---------------------+---------------------------------------------------------------------------+
+|       Center        |      layers/shape.json ->  "ef" -> effects/gFill.json ->  "s"             |
```
Comment (AnishGG) [inline docs/projects/lottie/layers/radial_gradient.rst:15]: No path effects/gFill.json

File: docs/projects/lottie/layers/radial_gradient.rst [hunk 13]
```diff
@@ -0,0 +1,28 @@
...
++---------------------+---------------------------------------------------------------------------+
+|       Radius        |      layers/shape.json ->  "ef" -> effects/gFill.json ->  "e"             |
```
Comment (AnishGG) [inline docs/projects/lottie/layers/radial_gradient.rst:17]: same

File: docs/projects/lottie/supported.rst [hunk 14]
```diff
@@ -362,6 +362,50 @@ Switch Group Layer
...
+Linear Gradient Layer
+---------------------
++---------------------+---------------------+---------------------------------------------+
+|       Property      |        Value        |               Type(Animation)               |
++=====================+=====================+=============================================+
+|       Z depth       |        |tick|       |                  |cross|                    |
++---------------------+---------------------+---------------------------------------------+
+|       Opacity       |        |tick|       |                  |cross|                    |
++---------------------+---------------------+---------------------------------------------+
+|     Blend_method    |        |cross|      |                  |cross|                    |
```
Comment (AnishGG) [inline docs/projects/lottie/supported.rst:374]: Please check other supported features and use "Partially supported" and a "dash" for consistency.

File: docs/projects/lottie/supported.rst [hunk 15]
```diff
@@ -362,6 +362,50 @@ Switch Group Layer
...
++---------------------+---------------------+---------------------------------------------+
+|       Point 1       |        |tick|       |                  |tick|                     |
```
Comment (AnishGG) [inline docs/projects/lottie/supported.rst:376]: No point1 param

File: docs/projects/lottie/supported.rst [hunk 16]
```diff
@@ -362,6 +362,50 @@ Switch Group Layer
...
++---------------------+---------------------+---------------------------------------------+
+|       Point 2       |        |tick|       |                  |tick|                     |
```
Comment (AnishGG) [inline docs/projects/lottie/supported.rst:378]: No point2 param

File: docs/projects/lottie/supported.rst [hunk 17]
```diff
@@ -362,6 +362,50 @@ Switch Group Layer
...
+Radial Gradient Layer
+---------------------
++---------------------+---------------------+---------------------------------------------+
+|       Property      |        Value        |               Type(Animation)               |
++=====================+=====================+=============================================+
+|       Z depth       |        |tick|       |                  |cross|                    |
++---------------------+---------------------+---------------------------------------------+
+|       Opacity       |        |tick|       |                  |cross|                    |
++---------------------+---------------------+---------------------------------------------+
+|     Blend_method    ||cross|              |                  |cross|                    |
```
Comment (AnishGG) [inline docs/projects/lottie/supported.rst:396]: Same as linear gradient's comment

**Comment by saurabh-ramola** [comment]
@AnishGG , I have corrected the formatting and added some more information as requested by you. Let me know if I have missed out on anything else ^__^

**Comment by AnishGG** [comment]
@saurabh-ramola can you resolve the conflicts.

**Comment by saurabh-ramola** [comment]
> @saurabh-ramola can you resolve the conflicts.

Done ^__^

**Comment by ice0** [comment]
Looks like conflict again :(

**Comment by saurabh-ramola** [comment]
> Looks like conflict again :(

Corrected it now

**Comment by AnishGG** [comment]
Great!
@ice0 seems ready for merging? ^__^

**Comment by ice0** [comment]
Merged. Thank you!

**Pull request merged**